tests/bsim/bluetooth/host/misc/sample_test/src/main.c

```diff
@@ -9,11 +9,33 @@
 #include "bs_tracing.h"
 #include "bstests.h"
 #include "babblekit/testcase.h"
+#include "testlib/log_utils.h"
+
 
 extern void entrypoint_dut(void);
 extern void entrypoint_peer(void);
 extern enum bst_result_t bst_result;
 
+static uint8_t	set_log_level_run_time = 3;
+
+static void test_args(int argc, char *argv[])
+{
+	size_t argn = 0;
+	const char *arg = argv[argn];
+
+	if(strcmp(arg, "log_level") == 0){
+		set_log_level_run_time = strtoul(argv[++argn], NULL, 10);
+
+		TEST_PRINT("arg : %d\n", set_log_level_run_time);
+
+		if(set_log_level_run_time >= 0 && 
+			set_log_level_run_time <= 4){
+			bt_testlib_log_level_set("dut", set_log_level_run_time);
+		}
```

Comment:
We should abort if an invalid log value is set

Reply:
can i set to default in that case? setting `LOG_LEVEL_INF` if the `arg` is out of bounds

Reply:
`TEST_PRINT("Invalid arguments to set log level: %d", runtime_log_level);` printing out as such?

Reply:
It says device `p_2G4: WARNING: (src/bs_pc_base.c:408): Device 0 left the party unsuspectingly.. I treat it as if it disconnected` when I call `exit(1)` when invalid arguments are received. Is it good?

Reply:
> It says device `p_2G4: WARNING: (src/bs_pc_base.c:408): Device 0 left the party unsuspectingly.. I treat it as if it disconnected` when I call `exit(1)` when invalid arguments are received. Is it good?

The message from the phy (p_2G4) is just telling that the device disappeared from the simulation without properly disconnecting.
You should not call `exit()`. You can instead call, for ex., ```TEST_FAIL("some message");```